app/lib/submission_builder/ty2024/states/nj/documents/nj_w2.rb

```diff
@@ -12,10 +12,10 @@ def document
 
               xml_node.at("EmployerNameControlTxt").name = "EmployerNameControl"
               xml_node.at("ControlNum").name = "ControlNumber" if xml_node.at("ControlNum").present?
-              xml_node.at("EmployeeNm").name = "EmployeeName"
+              xml_node.at("EmployeeNm").name = "EmployeeName" if xml_node.at("EmployeeNm").present?
               xml_node.at("PriorUSERRAContributionYr").name = "PriorYearUserraContribution" if xml_node.at("PriorUSERRAContributionYr").present?
               xml_node.at("OtherDeductionsBenefitsGrp").name = "OtherDeductsBenefits" if xml_node.at("OtherDeductionsBenefitsGrp").present?
-              xml_node.at("EmployerStateIdNum").name = "EmployersStateIdNumber"
+              xml_node.at("EmployerStateIdNum").name = "EmployersStateIdNumber" if xml_node.at("EmployerStateIdNum").present?
```

Comment:
[dust] this makes me wonder -- are there any XML attributes that we actually do want to fail on (in a caught way of course, not just 500) because it would mean something is wrong in the input?

Reply:
that makes sense to me 👍